pages/account/logout.vue

```diff
@@ -8,9 +8,9 @@
     layout: 'minimal',
     middleware: [
       'auth',
-      ({ $auth, store }) => {
-        store.dispatch('set/reset');
-        $auth.logout();
+      ({ app }) => {
```

Comment:
Unless there is a particular reason not to, use the $goto helper:
```diff
diff --git a/pages/account/logout.vue b/pages/account/logout.vue
index f107d5aa..5fc6c13a 100644
--- a/pages/account/logout.vue
+++ b/pages/account/logout.vue
@@ -8,9 +8,14 @@
     layout: 'minimal',
     middleware: [
       'auth',
-      ({ app }) => {
-        const path = app.$auth.strategies.keycloak.options.end_session_endpoint;
-        window.location.replace(`${path}?redirect_uri=${encodeURIComponent(window.location.origin)}`);
+      ({ $auth, store, app }) => {
+        store.dispatch('set/reset');
+        app.$goto({
+          path: $auth.strategies.keycloak.options.end_session_endpoint,
+          query: {
+            'redirect_uri': window.location.origin
+          }
+        });
       }
     ]
   };
```

Reply:
@rwd The $goto helper uses "window.location.href" for redirection. Like I mentioned in this commit
https://github.com/europeana/portal.js/pull/793/commits/e1b6ea38de382990b7f8d4dbf6ed6b73bb7b2415
there is a compatibility issue with the href property of window.location, that's why I replaced it with the method window.location.replace() which works also with Safari.

Please let me know how I should proceed. Maybe I could adjust the $goto method to use window.location.replace instead?